examples/interactive/cli-platform/Env.roc

```diff
@@ -13,3 +13,90 @@ cwd =
             Ok (InternalPath.fromArbitraryBytes bytes)
 
     InternalTask.fromEffect effect
+
+# ## Sets the [current working directory](https://en.wikipedia.org/wiki/Working_directory)
+# ## in the environment. After changing it, file operations on relative [Path]s will be relative
+# ## to this directory.
+# setCwd : Path -> Task {} [InvalidCwd]* [Env]*
+# setCwd = InternalTask.fromEffect Effect.setCwd
+# ## Gets the path to the currently-running executable.
+# exePath : Task Path [ExePathUnavailable]* [Env]*
+# exePath = InternalTask.fromEffect Effect.setCwd
+## Reads the given environment variable.
+##
+## If the value is invalid Unicode, the invalid parts will be replaced with the
+## [Unicode replacement character](https://unicode.org/glossary/#replacement_character)
+## (`�`).
+var : Str -> Task Str [VarNotFound]* [Env]*
+var = \name ->
+    Effect.envVar name
+    |> Effect.map (\result -> Result.mapErr result \{} -> VarNotFound)
+    |> InternalTask.fromEffect
+
+# ## Reads the given environment variable and attempts to decode it.
+# ##
+# ## The type being decoded into will be determined by type inference. For example,
+# ## if this ends up being used like a `Task U16 …` then the environment variable
+# ## will be decoded as a string representation of a `U16`.
+# ##
+# ##     getU16Var : Str -> Task U16 [VarNotFound, DecodeErr DecodeError]* [Env]*
+# ##     getU16Var = \var -> Env.decode var
+# ##     # If the environment contains a variable NUM_THINGS=123, then calling
+# ##     # (getU16Var "NUM_THINGS") would return a task which succeeds with the U16 number 123.
+# ##     #
+# ##     # However, if the NUM_THINGS environment variable was set to 1234567, then
+# ##     # (getU16Var "NUM_THINGS") would fail because that number is too big to fit in a U16.
+# ##
+# ## Supported types:
+# ## - strings
+# ## - numbers, as long as they contain only numeric digits, up to one `.`, and an optional `-` at the front for negative numbers
+# ## - comma-separated lists (of either strings or numbers), as long as there are no spaces after the commas
```

Comment:
Based on our discussion earlier today, we'll want to try to support arbitrary decoding types here, yeah?